packages/near-fast-auth-signer-e2e-tests/test-app/App.tsx

```diff
@@ -74,7 +74,7 @@ export default function App() {
       )}
       <button
         type="button"
-        data-test-id="sign-transaction-button"
+        data-testid="sign-transaction-button"
```

Comment:
this is good fix, noticed that we have lots of places need to be fixed. (Should address in different PR)

Reply:
Both are ok to use but I think we should standardize one across the codebase

Reply:
Yeah, we need a standard. I started cleaning up to use the Playwright way, that's `data-testid`.